docs/getting-started-guides/kubectl.md

```diff
@@ -0,0 +1,63 @@
+---
+---
+
+<style>
+li>.highlighter-rouge {position:relative; top:3px;}
+</style>
+
+## Overview
+
+kubectl is the command line tool you will use when interacting with kubernetes.
+
+You should use a version of kubectl that is newer than your server.  `kubectl version` will print
+the server and client versions.
+
+## Download a release
+
+Download kubectl from the [official Kubernetes releases](https://console.cloud.google.com/storage/browser/kubernetes-release/release/):
+
+On MacOS:
+
+```
```

Comment:
You get better formatting if you use tick tick tick shell .... cannot type tick-tick-tick because it formats ...